docs/how-to-guides/develop-a-prompty/prompty-output-format.md

```diff
@@ -0,0 +1,231 @@
+# Prompty output format
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+In this doc, you will learn:
+- Understand how to handle output format of prompty like: text, json_object.
+- Understand how to consume stream output of prompty
+
+## Format prompty output
+
+### Text output
+By default, the prompty returns the message of first choices.
+
+```prompty
+---
+name: Text Format Prompt
+description: A basic prompt that uses the GPT-3 chat API to answer questions
+model:
+    api: chat
+    configuration:
+      type: azure_openai
+      connection: <connection_name>
+      azure_deployment: gpt-35-turbo-0125
+    parameters:
+      max_tokens: 128
+      temperature: 0.2
+sample:
+  inputs:
+    first_name: John
+    last_name: Doh
+    question: what is the meaning of life?
+---
+system:
+You are an AI assistant who helps people find information.
+As the assistant, you answer questions briefly, succinctly,
+and in a personable manner using markdown and even add some personal flair with appropriate emojis.
+
+# Safety
+- You **should always** reference factual statements to search results based on [relevant documents]
+- Search results based on [relevant documents] may be incomplete or irrelevant. You do not make assumptions
+# Customer
+You are helping {{first_name}} {{last_name}} to find answers to their questions.
+Use their name to address them in your responses.
+
+user:
+{{question}}
+```
+The return value of the prompty is a string content.
+```text
+Ah, the age-old question about the meaning of life! 🌍🤔 The meaning of life is a deeply philosophical and subjective topic. Different people have different perspectives on it. Some believe that the meaning of life is to seek happiness and fulfillment, while others find meaning in personal relationships, accomplishments, or spiritual beliefs. Ultimately, it's up to each individual to explore and discover their own purpose and meaning in life. 🌟
+```
+
+### Json object output
+When the user meets the following conditions, prompty returns content of first choices as a dict.
+- Define response_format to type: json_object in parameters
+- Specify the return json format in template.
+
+**Note**: response_format is compatible with GPT-4 Turbo and all GPT-3.5 Turbo models newer than gpt-3.5-turbo-1106. For more details, refer to this [document](https://platform.openai.com/docs/api-reference/chat/create#chat-create-response_format).
```

Comment:
add \` \` on model name like: `GPT-4 Turbo`

Reply:
fix it